variants/backend-base/example.env.tt

```diff
@@ -26,3 +20,13 @@ PORT=3000
 # SENTRY_DSN=http://public@example.com/project-id
 SENTRY_CSP_HEADER_REPORT_ENDPOINT=https://SOMECODE.ingest.sentry.io/api/SOMENUMS/security/?sentry_key=SOMETHING
 SENTRY_ENV=development
+
+# NEVER just copy these secrets from development env to a production env.  Run:
+#
+#     bin/rails db:encryption:init
+#
+# to create new versions of these secrets for each deployed environment (e.g.
+# staging, production)
+ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY="PLACEHOLDER_PRIMARY_KEY"
+ACTIVE_RECORD_ENCRYPTION_DETERMINISTIC_KEY="PLACEHOLDER_DETERMINISTIC_KEY"
+ACTIVE_RECORD_ENCRYPTION_KEY_DERIVATION_SALT="PLACEHOLDER_KEY_DERIVATION_SALT"
```

Comment:
These env vars need to exist to load Rails because we are loading them with `ENV.fetch`. We need to run Rails to set realistic values so we need to set these env vars to an initial placeholder value and later set better values.